nemo/collections/nlp/models/qa_model.py

```diff
@@ -45,59 +46,41 @@ def input_types(self) -> Optional[Dict[str, NeuralType]]:
     def output_types(self) -> Optional[Dict[str, NeuralType]]:
         return self.classifier.output_types
 
-    def __init__(
-        self,
-        num_classes: int = 2,
-        pretrained_model_name: str = 'bert-base-cased',
-        config_file: Optional[str] = None,
-        num_layers: int = 1,
-        activation: str = 'relu',
-        log_softmax: bool = False,
-        dropout: float = 0.0,
-        use_transformer_init: bool = True,
-    ):
-        """
-        Args:
-            num_classes: number of classes
-            pretrained_model_name: pretrained language model name, to see the complete list use
-            config_file: model config file
-            num_layers: number of fully connected layers in the multilayer perceptron (MLP)
-            activation: activation to usee between fully connected layers in the MLP
-            log_softmax: whether to apply softmax to the output
-            dropout: dropout to apply to the input hidden states
-            use_transformer_init: whether to use pre-trained transformer weights for weights initialization
-        """
-        # init superclass
-        # TODO: This is a workaround - please fix - see text_classification_model or asr for example
-        cfg = DictConfig(
-            {
-                'num_classes': num_classes,
-                'pretrained_model_name': pretrained_model_name,
-                'config_file': config_file,
-                'num_layers': num_layers,
-                'activation': activation,
-                'log_softmax': log_softmax,
-                'dropout': dropout,
-                'use_transformer_init': use_transformer_init,
-            }
+    def __init__(self, cfg: DictConfig, trainer: Trainer = None):
+        self.max_answer_length = cfg.max_answer_length
+        self.output_prediction_file = cfg.output_prediction_file
+        self.output_nbest_file = cfg.output_nbest_file
```

Comment:
could you move this to validation_ds yaml and delete member since this is validation data specific